docs/man/containerd-config.toml.5.md

```diff
@@ -13,14 +14,30 @@ does not exist at the appropriate location or is not provided via the
 **--config** option containerd uses its default configuration settings, which
 can be displayed with the **containerd config(1)** command.
 
+For the general CRI configuration users can refer to [`../cri/config.md`](../cri/config.md) and [`../cri/containerd-config-cri-ex.x.md`](../cri/containerd-config-cri-ex.x.md) for specific CRI examples
+
+For default configurations users can refer to [`../man/containerd-config.8.md`](../man/containerd-config.8.md) 
```

Comment:
Relative paths won't work with man pages

Reply:
Good to know, fixed and pushed, thx!